### ISSUE TITLE
RMSLE in documentation

### ISSUE THREAD

**Issue by banmeidareu** [dmlc/xgboost#5500]
Hi, 
In the documentation, it seems that RMSLE is defined without the summation around the square-log differences.

I believe it should include a summation around the square.
Examples below (in the part about rmsle):
https://github.com/dmlc/xgboost/blob/cd3a3f99da3869d00c82701f34eef026fc9b204a/doc/tutorials/custom_metric_obj.rst
https://github.com/dmlc/xgboost/blob/d0b86c75d9061b93ea6c19097cc690bea9e985ba/doc/parameter.rst

Not a problem in implementation as here it DOES have the sum:
https://github.com/dmlc/xgboost/blob/5b2f805e7474c1d18f0a5262f8964d49f0ce4d89/demo/guide-python/custom_rmsle.py

Sorry if this is the wrong place for documentation related stuff!

**Comment by hcho3** [comment]
The definition given in the tutorial is element-wise, i.e. it is defined in terms of one data point. The summation is introduced when we compute the loss over the training data points.
